src/order-history/selectors.js

```diff
@@ -0,0 +1,4 @@
+export const storeName = 'orderHistory';
```

Comment:
Mentioned this in a separate comment, but I'm not sure it's worth abstracting the name of the store key; we don't have a way of dynamically registering pages or anything anyway, so it's all sorta manually strung together at the moment anyway.

Reply:
I think this is important to keep one way or another. We shouldn't rely on a parent providing the correct name.

Reply:
Yeah, I won't go to the mat on this. :) 👍